samples/angular-todo/src/environments/environment.ts

```diff
@@ -1,8 +1,8 @@
 export const environment = {
-  AM_URL: 'https://openam-crbrl-01.forgeblocks.com/am',
+  AM_URL: 'https://openam-sdks.forgeblocks.com/am/',
   REALM_PATH: 'alpha',
   WEB_OAUTH_CLIENT: 'WebOAuthClient',
-  JOURNEY_LOGIN: 'Login',
+  JOURNEY_LOGIN: 'google-login-web',
```

Comment:
I'd really like to keep these values generic and/or use defaults found in a basic tenant. Could we switch these values back to generic or defaults?

Reply:
It could set it back, however it'd be a different journey with no option for IdPs.
thoughts??

Reply:
Yeah, let's set it back. We should always have the sample apps point to the default trees: `Login` and `Registration`. As we build out new features, and support new nodes/callbacks, we will continue to default the sample apps to these trees. We are just adding these features in order to support an expanded set of nodes _if_ the customer uses them when experimenting with the sample apps.